lib/npg_warehouse/loader.pm

```diff
@@ -1,46 +1,54 @@
 package npg_warehouse::loader;
 
 use Carp;
-use English qw{-no_match_vars};
+use namespace::autoclean;
 use Moose;
 use MooseX::StrictConstructor;
+use List::MoreUtils qw/uniq/;
+use Readonly;
+use Try::Tiny;
 
 use npg_warehouse::Schema;
 use npg_tracking::Schema;
 use npg_qc::Schema;
+use WTSI::DNAP::Warehouse::Schema;
 use npg_qc::autoqc::qc_store;
-
+use st::api::lims;
 use npg_warehouse::loader::autoqc;
 use npg_warehouse::loader::lims;
 use npg_warehouse::loader::qc;
 use npg_warehouse::loader::npg;
 use npg_warehouse::loader::run_status;
 
-use Readonly;
-## no critic (ProhibitLongChainsOfMethodCalls ProhibitExcessComplexity ProhibitNoisyQuotes)
-
-Readonly::Scalar our $FORWARD_END_INDEX   => 1;
-Readonly::Scalar our $REVERSE_END_INDEX   => 2;
-Readonly::Scalar our $PLEXES_KEY         => q[plexes];
+with 'MooseX::Getopt';
 
 our $VERSION = '0';
 
+Readonly::Scalar my $FORWARD_END_INDEX   => 1;
+Readonly::Scalar my $REVERSE_END_INDEX   => 2;
+Readonly::Scalar my $PLEXES_KEY          => q[plexes];
+Readonly::Scalar my $DEFAULT_LIMS_DRIVER => q[ml_warehouse_fc_cache];
+Readonly::Scalar my $TENTHOUSAND         => 10_000;
```

Comment:
_Maybe_ provide more meaning with constant name e.g. ```$MAX_RUN_WITH_NO_BATCH_ID```? If I've got that right....